frontend/src/metabase/core/components/Select/Select.jsx

```diff
@@ -206,57 +193,64 @@ export default class Select extends Component {
       .filter(n => n);
 
     return (
-      <PopoverWithTrigger
-        ref={ref => (this._popover = ref)}
-        triggerElement={
-          this.props.triggerElement || (
-            <SelectButton
-              ref={this.selectButtonRef}
-              className="flex-full"
-              hasValue={selectedNames.length > 0}
-              disabled={disabled}
-              {...buttonProps}
-            >
-              {selectedNames.length > 0
-                ? selectedNames.map((name, index) => (
-                    <span key={index}>
-                      {name}
-                      {index < selectedNames.length - 1 ? ", " : ""}
-                    </span>
-                  ))
-                : placeholder}
-            </SelectButton>
-          )
-        }
-        onClose={composeEventHandlers(onClose, this.handleClose)}
+      <TippyPopoverWithTrigger
+        placement="bottom-start"
+        onHide={onClose}
```

Comment:
Was previously `onClose`. See the comment I made on the previous change at line#230. I'm not sure if this would be a problem, but from initial testing, this seems okay. Do you think `TippyPopoverWithTrigger` should accept `onClose` given how it was being used? FYI @daltojohnso